modules/caddyhttp/reverseproxy/httptransport.go

```diff
@@ -564,6 +568,21 @@ func (t TLSConfig) MakeTLSClientConfig(ctx caddy.Context) (*tls.Config, error) {
 		cfg.RootCAs = rootPool
 	}
 
+	if t.CARaw != nil {
+		if len(t.RootCAPool) > 0 || len(t.RootCAPEMFiles) > 0 {
+			return nil, fmt.Errorf("Conflicting config for Root CA pool")
+		}
+		caRaw, err := ctx.LoadModule(t, "CARaw")
+		if err != nil {
+			return nil, fmt.Errorf("failed to load ca module: %v", err)
+		}
+		ca, ok := caRaw.(caddytls.CA)
+		if !ok {
+			return nil, fmt.Errorf("CARaw module '%s' is not a certificate pool provider", ca)
```

Comment:
Use `ca` instead of `CARaw` because the user does not know the field name inside the struct.

Reply:
I am assuming you are talking about the error message here right? If so, then this should now be resolved. Also, I think you are using the same error message when provisioning the `ClientAuthentication` module inside `modules/caddytls/connpolicy.go`, so maybe we should also change that? 

Sorry in advance if it's not about the error message and if  I misinterpreted your comment 😅

Reply:
> so maybe we should also change that?

Yep. It was probably an auto-complete mistake, and we missed it.